lib/src/HalstackContext.tsx

```diff
@@ -276,13 +286,13 @@ const parseTheme = (theme) => {
   wizardTokens.visitedHelperTextFontColor = theme?.wizard?.fontColor ?? wizardTokens.visitedHelperTextFontColor;
   wizardTokens.selectedHelperTextFontColor = theme?.wizard?.fontColor ?? wizardTokens.selectedHelperTextFontColor;
   wizardTokens.unvisitedStepBorderColor =
-    setOpacity(theme?.wizard?.fontColor, 0.6) ?? wizardTokens.unvisitedStepBorderColor;
+    addLightness(theme?.wizard?.fontColor, 80) ?? wizardTokens.unvisitedStepBorderColor;
   wizardTokens.unvisitedStepFontColor =
-    setOpacity(theme?.wizard?.fontColor, 0.6) ?? wizardTokens.unvisitedStepFontColor;
+    addLightness(theme?.wizard?.fontColor, 80) ?? wizardTokens.unvisitedStepFontColor;
   wizardTokens.unvisitedLabelFontColor =
-    setOpacity(theme?.wizard?.fontColor, 0.6) ?? wizardTokens.unvisitedLabelFontColor;
+    addLightness(theme?.wizard?.fontColor, 80) ?? wizardTokens.unvisitedLabelFontColor;
   wizardTokens.unvisitedHelperTextFontColor =
-    setOpacity(theme?.wizard?.fontColor, 0.6) ?? wizardTokens.unvisitedHelperTextFontColor;
+    addLightness(theme?.wizard?.fontColor, 80) ?? wizardTokens.unvisitedHelperTextFontColor;
```

Comment:
We should add 40% lightness instead of 80% to get the correct value. This applies to the following tokens of this component:
unvisitedStepFontColor, unvisitedLabelFontColor, and unvisitedHelperTextFontColor.

Reply:
Done!